### PR TITLE
Suppress the expected errors from Select-Xml tests

### PR DESCRIPTION
**Pull request by daxian-dbw**
Suppress expected errors from `Select-Xml` tests to reduce noises:
```
Describing XML cmdlets
   Context Select-XML
++++++
Select-XML : The file 'xmltestfile' cannot be read: Could not find file 'C:\projects\powershell-f975h\xmltestfile'.
At C:\projects\powershell-f975h\test\powershell\Modules\Microsoft.PowerShell.Utility\xml.tests.ps1:68 char:17
+                 Select-XML @params "Root" -ErrorVariable err
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (xmltestfile:String) [Select-Xml], ArgumentException
+ FullyQualifiedErrorId : ProcessingFile,Microsoft.PowerShell.Commands.SelectXmlCommand
 
+
Select-XML : Cannot open the file because the current provider (Microsoft.PowerShell.Core\Environment) cannot open files.
At C:\projects\powershell-f975h\test\powershell\Modules\Microsoft.PowerShell.Utility\xml.tests.ps1:68 char:17
+                 Select-XML @params "Root" -ErrorVariable err
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (env:xmltestfile:String) [Select-Xml], InvalidOperationException
+ FullyQualifiedErrorId : ProcessingFile,Microsoft.PowerShell.Commands.SelectXmlCommand
 
...
```
After the fix:
```
Describing XML cmdlets
   Context Select-XML
    [+] literalpath with relative paths 591ms
    [+] literalpath with absolute paths 82ms
    [+] literalpath with path with dots 11ms
    [+] path with relative paths 11ms
    [+] path with absolute paths 11ms
    [+] path with path with dots 13ms
    [+] Verifies a non filesystem path using literalPath should fail 46ms
    [+] Verifies a non filesystem path using path should fail 9ms
    [+] Invalid xml file 72ms
    [+] -xml works with inputstream 46ms
    [+] Returns error for invalid xmlnamespace 16ms
    [+] Returns error for invalid content 18ms
    [+] ToString() works correctly on nested node 15ms
    [+] ToString() works correctly with file 64ms
Tests completed in 1.01s
Passed: 14 Failed: 0 Skipped: 0 Pending: 0
```

### CODE REVIEW

**Review by adityapatwardhan — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@daxian-dbw Can you have a look at the failures?

**Comment by daxian-dbw** [comment]
All failures are know issues:
Linux CI failure --> #5567
macOS CI failure --> #5590

File: test/powershell/Modules/Microsoft.PowerShell.Utility/xml.tests.ps1 [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
         BeforeAll {
             $fileName = New-Item -Path 'TestDrive:\testSelectXml.xml'
-	        Push-Location "$fileName\.."
-	        "<Root>" | out-file -LiteralPath $fileName
-	        "   <Node Attribute='blah' />" | out-file -LiteralPath $fileName -Append
-	        "</Root>" | out-file -LiteralPath $fileName -Append
+            Push-Location "$fileName\.."
+            "<Root>" | out-file -LiteralPath $fileName
+            "   <Node Attribute='blah' />" | out-file -LiteralPath $fileName -Append
+            "</Root>" | out-file -LiteralPath $fileName -Append
```
Comment (iSazonov) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/xml.tests.ps1]: Tabs?

**Review by iSazonov — APPROVED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Utility/xml.tests.ps1 (hunk 1 above)
Comment (daxian-dbw) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/xml.tests.ps1]: It used to be tabs, and now is changed to spaces as in rest of the file.

**Comment by adityapatwardhan** [comment]
@daxian-dbw Restarted Travis-CI.

**Comment by daxian-dbw** [comment]
@adityapatwardhan CI builds have passed, can you please merge this PR?

**Pull request merged**